server/routes/journeys/record-decision/check-answers/controller.ts

```diff
@@ -12,6 +14,37 @@ export class DecisionCheckAnswersController extends BaseJourneyController {
     })
   }
 
+  checkSubmitToAPI = async (req: Request, res: Response, next: NextFunction) => {
+    const decision = req.journeyData.decisionAndActions!
+    try {
+      await this.csipApiService.createDecision(req, {
+        date: todayString(),
+        recordedBy: res.locals.user.username,
+        recordedByDisplayName: res.locals.user.displayName,
+        conclusion: decision.conclusion!,
+        outcomeTypeCode: decision.outcome!.code,
+        signedOffByRoleCode: decision.signedOffByRole!.code,
+        actions: [],
+        actionOther: decision.actionOther!,
```

Comment:
actionOther is optional field and can be null, can't `!` on it.
and we miss `nextSteps`